src/Plugin/SwiftMailerPlugin/ClearSpools.php

```diff
@@ -39,5 +42,8 @@ public function cleanUp()
                 $this->logger->error(sprintf('Exception occurred while flushing email queue: %s', $exception->getMessage()));
             }
         }
+        foreach ($this->transports as $transport) {
```

Comment:
Why stop all the transports if we only use one of the transports? 
* https://github.com/LongRunning/LongRunning/blob/master/src/Plugin/SwiftMailerPlugin/ClearSpools.php#L37